### ISSUE TITLE
Missing localization in production

### ISSUE THREAD

**Issue by flodolo** [mozilla/send#1127]
At least one locale (hr) is missing from production.

I'm not exactly sure how we're checking for locale completion, and if we are. 

But I realized someone force pushed the repository's master a while ago (I can't tell when, because I reset the repository before checking), breaking my own tracking system. In general, that's a really bad idea for a repository with forks in the hundreds.

**Comment by pdehaan** [comment]
**TL;DR:** I think these are the current 100% locales that are NOT currently in [package.json](https://github.com/mozilla/send/blob/779fe94550eeec00c35a1bfeeb3d1493ccdcfd7b/package.json#L132-L184):

- "en-GB"
- "hr"

---

```
pontoonql 100% [ "ar",        "az", "bs", "ca", "cak", "cs", "cy", "da", "de", "dsb", "el", "en-CA", "en-GB", "es-AR", "es-CL", "es-ES", "es-MX", "et", "fa", "fr", "fy-NL", "hr", "hsb", "hu", "ia", "id", "it", "ja", "ka", "kab", "ko", "ms", "nb-NO", "nl", "nn-NO", "pt-BR", "pt-PT", "ro", "ru", "sk", "sl", "sq", "sr", "sv-SE",       "tl", "tr", "uk", "vi", "zh-CN", "zh-TW" ]
package.json   [ "ar", "ast", "az", "bs", "ca", "cak", "cs", "cy", "da", "de", "dsb", "el", "en-CA",          "es-AR", "es-CL", "es-ES", "es-MX", "et", "fa", "fr", "fy-NL",       "hsb", "hu", "ia", "id", "it", "ja", "ka", "kab", "ko", "ms", "nb-NO", "nl", "nn-NO", "pt-BR", "pt-PT", "ro", "ru", "sk", "sl", "sq", "sr", "sv-SE", "te", "tl", "tr", "uk", "vi", "zh-CN", "zh-TW" ]
```

You can see a few gaps... Looks like "en-GB" and "hr" are both at [100% translated](https://pontoonql.now.sh/test-pilot-firefox-send/100) strings, but aren't in package.json.

Looks like we have a few other locales which are in the package.json, but not 100% [anymore]:

- "ast" (98.9%)
- "fi" (98.9%) (oops, not currently in package.json)
- "te" (91.6%)

```sh
$ npx pdehaan/pontoonql test-pilot-firefox-send 80 | jq 'sort_by(.locale.code) | [.[] | {code: .locale.code, progress: .progress}]'

# I manually removed all the results which were at 100%...
[
  {
    "code": "ast",
    "progress": 98.94736842105263
  },
  {
    "code": "fi",
    "progress": 98.94736842105263
  },
  {
    "code": "te",
    "progress": 91.57894736842105
  }
]
```

**Comment by johngruen** [comment]
As of last deploy these weren't 100%...we'll pick this up on the next deployment

**Comment by dannycoates** [comment]
fixed in 6ef9b7be92dbfab3988ff92d0430fc01579f447b
